projects/ngrx-auto-entity/src/lib/operators.ts

```diff
@@ -74,18 +74,21 @@ export const handleError = <TModel, TErrorAction>(
   if (error.err instanceof TypeError) {
     const serviceName = `${pascalCase(error.info.modelName)}Service`;
     console.error(
-      `[NGRX-AE] ! NgRxAutoEntityService Error: Unable to locate load method in the ${serviceName}`,
+      `[NGRX-AE] ! NgRxAutoEntityService Error: Unable to locate load method in the ${serviceName}`,,
```

Comment:
double `,`.

I think it produces a SyntaxError :thinking:

Reply:
Woops, copy paste mishap. Should be fixed now.